pombola/core/views.py

```diff
@@ -325,7 +325,7 @@ def position(request, pt_slug, ok_slug=None, o_slug=None):
     # Order by place name unless ordering by person name is requested:
     order = 'place'
     if request.GET.get('order') == 'name':
-        order = 'person__legal_name'
+        order = 'person__sort_name'
```

Comment:
`sort_name` is obviously the right thing, but it can be blank? Can we rely on the data here?